src/Containers/packaging/build/Microsoft.NET.Build.Containers.props

```diff
@@ -15,6 +15,7 @@
 
     <!--Register our custom task-->
     <UsingTask TaskName="$(MSBuildThisFileName).Tasks.CreateNewImage" AssemblyFile="$(ContainerCustomTasksAssembly)"/>
+    <UsingTask TaskName="$(MSBuildThisFileName).Tasks.CreateImageIndex" AssemblyFile="$(ContainerCustomTasksAssembly)"/>
```

Comment:
@surayya-MS have you tried publishing a container from VS? I worry that this using task will only load a .NET version of the Task (even if the CreateImageIndex Task is never called from the single-RID VS publishing workflow).

Reply:
I tried it out and it doesn't work. Details:
1. I added Docker support to the project via VS UI, and it added `RuntimeIdentifiers` to my project. 
2. When I ran the project via VS UI, I got error
```Severity	Code	Description	Project	File	Line	Suppression State	Details
Error (active)	MSB4006	There is a circular dependency in the target dependency graph involving target "Publish". Since "_CopyAspNetCoreFilesToIntermediateOutputPath" has "DependsOn" dependence on "Publish", the circular is "Publish<-_CopyAspNetCoreFilesToIntermediateOutputPath<-CorePublish<-DotNetPublish<-_PublishBuildAlternative<-Publish".	container-webapp	C:\Program Files\dotnet\sdk\8.0.405-dev\Sdks\Microsoft.NET.Sdk.Publish\targets\CopyTargets\Microsoft.NET.Sdk.Publish.CopyFiles.targets	81		
```
3. I figured out it might be the `Publish` target that we add to the `MSBuild` task before `_PublishSingleContainer`. I removed `Publish` target to test my theory, and got another error saying that the image {my-app-name}:latest was not found. The image was created with another tag `latest-linux-x64`. So, VS relies on the tag being `latest`.

We need extra work on VS side to support multi-arch publishing

Reply:
As discussed, added check for `BuildingInsideVisualStudio` to call `_PublishSingleContainer` for fixing publish container from VS. Manually tested